libats-messaging/src/main/scala/com/advancedtelematic/libats/messaging/kafka/JsonDeserializer.scala

```diff
@@ -27,7 +27,9 @@ class JsonDeserializer[T](decoder: Decoder[T]) extends Deserializer[T] {
 
     msgXor match {
       case Right(v) => v
-      case Left(ex) => throw new JsonDeserializerException(s"Could not parse msg from $topic: ${ex.getMessage}")
+      case Left(ex) =>
+        _logger.error(s"Could not parse msg from $topic", ex)
+        null.asInstanceOf[T]
```

Comment:
We should add a flag to this serializer to throw instead of log, we might need it for some topics. You could add it to the constructor for now, default would be to skip it.